apps/web/app/_hooks/usePostMessageToRN.ts

```diff
@@ -0,0 +1,17 @@
+import { type WebviewMessageType } from "@ui/src/types/WebviewMessageTypes";
+import { useIsReactNativeWebview } from "./useIsReactNativeWebview";
+
+export interface WebViewMessage<T> {
+  type: WebviewMessageType;
+  data: T | null;
+}
+
+export const usePostMessageToRN = () => {
+  const isReactNativeWebview = useIsReactNativeWebview();
+
+  return <T>(message: WebViewMessage<T>) => {
```

Comment:
```suggestion
  return <T>(enevt: WebViewMessage<T>) => {
```
이벤트라고 표헌히는게 더 명확할거같기도 하고...

Reply:
이건 나도 애매해서 잘 모르긴 하겠음